tests/cases/integration/MigrationsExtension.phpt

```diff
@@ -69,11 +69,24 @@ class MigrationsExtensionTest extends TestCase
 	}
 
 
+	public function testDynamicContainerParameters()
+	{
+		if (!method_exists('Nette\DI\Compiler', 'setDynamicParameterNames')) {
+			\Tester\Environment::skip('Required Nette >= 2.4.7');
+		}
+		Assert::noError(function() {
+			$this->createContainer('dynamicParameters', [
+				'rootDir' => __DIR__,
+			]);
+		});
+	}
+
+
 	/**
 	 * @param  string $config
 	 * @return Nette\DI\Container
 	 */
-	protected function createContainer($config)
+	protected function createContainer($config, array $dynamicParameters = [])
```

Comment:
Default value for container parameters via constructor is `[]`. Should I really change it?